etc/scripts/install-on-okd.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+function enable_admission_webhooks {
+
+  echo "API_SERVER=$API_SERVER"
+  echo "KUBE_SSH_USER=$KUBE_SSH_USER"
+  echo "KUBE_SSH_KEY=$KUBE_SSH_KEY"
+  
+  ssh $KUBE_SSH_USER@$API_SERVER -i $KUBE_SSH_KEY /bin/bash << "EOF"
+  sudo su - root
+  cp -n /etc/origin/master/master-config.yaml /etc/origin/master/master-config.yaml.backup
+  cp -p /etc/origin/master/master-config.yaml /etc/origin/master/master-config.yaml.prepatch
+  cat > /etc/origin/master/master-config.patch << EOT
+admissionConfig:
+  pluginConfig:
+    MutatingAdmissionWebhook:
+      configuration:
+        apiVersion: apiserver.config.k8s.io/v1alpha1
+        kubeConfigFile: /dev/null
+        kind: WebhookAdmission
+    ValidatingAdmissionWebhook:
+      configuration:
+        apiVersion: apiserver.config.k8s.io/v1alpha1
+        kubeConfigFile: /dev/null
+        kind: WebhookAdmission
+EOT
+
+  oc ex config patch /etc/origin/master/master-config.yaml.prepatch -p "$(cat /etc/origin/master/master-config.patch)" > /etc/origin/master/master-config.yaml
+  /usr/local/bin/master-restart api && /usr/local/bin/master-restart controllers
+
+  rm /etc/origin/master/master-config.yaml.prepatch /etc/origin/master/master-config.patch
+  
+EOF
+
+   timeout_positive 60 "oc login -u system:admin"
+}
+
+# Installs OLM first, and then istio and knative using OLM operators
+
+if [ "$1" != "-q" ]; then
+  echo
+  echo "  WARNING: This script will blindly attempt to enable admission webhooks and"
+  echo "  install OLM, istio, and knative on your OKD cluster, so if any are already there,"
+  echo "  hijinks will ensue."
+  echo "  In order for admission webhooks to be enabled, you need to set \$KUBE_SSH_KEY variable," 
+  echo "  pointing to private SSH key for the cloud-user account."
+  echo
+  echo "  If your cluster is minishift, run $(dirname $0)/install-on-minishift.sh instead."
+  echo
+  echo "  Pass -q to disable this warning"
+  echo
+  read -p "Enter to continue or Ctrl-C to exit: "
+fi
+
+if [ -z "$KUBE_SSH_KEY" ]
+then
+      echo "\$KUBE_SSH_KEY is empty - aborting."
+      exit 1
+fi
+
+set -x
+
+API_SERVER=$(oc config view --minify | grep server | awk -F'//' '{print $2}' | awk -F':' '{print $1}')
+KUBE_SSH_USER="cloud-user"
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+
+source "$DIR/installation-functions.sh"
+
+enable_admission_webhooks
```

Comment:
Everything below this line seems the same as `install.sh`, so I'd rather invoke that here, as the minishift script does, e.g. `"$DIR/install.sh" -q`